cognite/powerops/custom_modules/power_model_v1/data_models/containers/ModelTemplate.container.yaml

```diff
@@ -37,15 +37,14 @@ properties:
     autoIncrement: false
     name: model
     description: The shop model file to use as template before mapping
-  source:
+  extraFiles:
     type:
-      list: false
-      collation: ucs_basic
-      type: text
+      list: true
+      type: file
     nullable: true
     autoIncrement: false
-    name: source
-    description: The source of the model
+    name: extraFiles
+    description: Extra files to include in the model
```

Comment:
I guess this is somewhat overlapping with the files (in particular reservoirMapping and cutOrderFile) on Case, but it can be good to be able to specify that "statically" here and per run on the Case type.
What about the `cogshop_files_config` file we discussed yesterday? I thought we agreed to add that here on the ModelTemplate